src/BlazorWebView/src/Maui/iOS/IOSWebViewManager.cs

```diff
@@ -182,10 +186,8 @@ private static void PresentAlertController(
 				if (cancelAction != null)
 					AddCancelAction(controller, () => cancelAction(controller));
 
-#pragma warning disable CA1416, CA1422 // TODO:  'UIApplication.Windows' is unsupported on: 'ios' 15.0 and later
-				GetTopViewController(UIApplication.SharedApplication.Windows.FirstOrDefault(m => m.IsKeyWindow)?.RootViewController)?
+				GetTopViewController(UIApplication.SharedApplication.ConnectedScenes.OfType<UIWindowScene>().SelectMany(scene => scene.Windows).FirstOrDefault(window => window.IsKeyWindow)?.RootViewController)?
 					.PresentViewController(controller, true, null);
-#pragma warning restore CA1416, CA1422
```

Comment:
Does this work if there are no scenes? We have this method: https://github.com/dotnet/maui/blob/f65f1eaad38b7e41d352383479f54b410d637c75/src/Core/src/Platform/iOS/UIApplicationExtensions.cs#L47

We should probably use that as it correctly handles iOS versions and scenes.